README.md

```diff
@@ -0,0 +1,67 @@
+# Altmetric Explorer API Client
+
+A basic API client for extracting data from Altmetric Explorer using the Explorer
+API ([documentation here](https://www.altmetric.com/explorer/documentation/api)).  This is intended to be a starter for 10 to help you get going with the API
+and you are encouraged to build out whatever you need from here.
+
+A quick example:
+
+```python
+from altmetric.explorer import api
+
+# Get real values at https://www.altmetric.com/explorer/settings
+API_KEY = 'abababababababab'
+API_SECRET = 'cdcdcdcdcdcdcdcd'
+
+client = api.Client('https://www.altmetric.com/explorer/api', API_KEY, API_SECRET)
+response = client.get_mention_sources()
+
+# response.meta returns the meta.response JSON object as a python dict
+print(response.meta)
+
+# response.data() returns a python generator that yields one row of data at
+# a time.  Paging is handled behind the scenes.
+for item in response.data:
+  print(item)
+```
+
+See the code in [examples/](examples/) for more examples.  You can run them by
```

Comment:
Does this still exist?  I'm struggling to find it

Reply:
Hmm there's a file called `example.py` in the root, this will need to be moved into a new "package" called `examples` for this documentation to apply.

Reply:
Oops.  No the Jupyter notebook replaced the examples.

Reply:
Fixed